packages/datagateway-download/src/downloadConfirmation/downloadConfirmDialog.component.tsx

```diff
@@ -109,16 +105,15 @@ interface DownloadConfirmDialogProps
   isTwoLevel: boolean;
   open: boolean;
 
-  // TODO: pass in the function to call to redirect to the status tab.
-  // setStatus: () => void;
+  setStatus: () => void;
```

Comment:
This `setStatus` is a pretty confusing name - it took me reading the removed comments to realise what it is doing. Perhaps something like `redirectToStatusTab`?